docs/plugins/automelt.rst

```diff
@@ -6,13 +6,41 @@ automelt
     :tags: untested fort productivity items stockpiles
     :no-command:
 
-When `enabled <enable>`, this plugin adds an option to the :kbd:`q` menu for
-stockpiles. When the ``automelt`` option is selected for the stockpile, any
-items placed in the stockpile will automatically be designated to be melted.
+Automelt checks monitor-enabled stockpiles once every in-game day, and will mark valid items for melting.
+
+Please see `gui/automelt` for the interactive configuration dialog.
 
 Usage
 -----
 
 ::
 
     enable automelt
+    automelt [status]
+    automelt (designate)
+    automelt (monitor|nomonitor) <stockpile>
+
+Examples
+--------
+
+Automatically designate all meltable items in the stockpile ("melt") for melting. ::
+
+    enable automelt
+    automelt monitor melt
+
+Enable monitoring for the stockpile ("melt"), and mmediately designate all meltable items in monitored stockpiles for melting. ::
+
+    automelt monitor melt
```

Comment:
I think that adding a stockpile to the monitor list should immediately designate its contents for some immediate feedback, no second command necessary.

Reply:
Added in latest